### PR TITLE
feat(cellguide): add boilerplate for docker image

### PR DESCRIPTION
**Pull request by atarashansky**
## Reason for Change

- #TICKET_NUMBER
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

- add
- remove
- modify

## Testing steps

- Either list QA steps or reasoning you feel QA is unnecessary
- Describe how you made sure to know that your changes worked. Should allow someone else to go verify your code without in depth knowledge.
- "Unit tested only", "tested in rdev by a, b, c, verifying feature worked by... ", "manually ran pipeline locally with these results: ..."

### CODE REVIEW

File: .github/workflows/push-tests.yml [hunk 1]
```diff
@@ -312,7 +312,7 @@ jobs:
         if: steps.changed-files.outputs.any_changed == 'true'
         run: |
           echo "DOCKER_REPO=${DOCKER_REPO}" > .env.ecr
-          make local-rebuild-processing
+          make local-rebuild-wmg-processing
```
Comment (atarashansky) [inline .github/workflows/push-tests.yml:315]: This was a bug in the WMG processing push tests. It was rebuilding the wrong image

File: Dockerfile.cellguide_pipeline [hunk 2]
```diff
@@ -0,0 +1,30 @@
+ARG BASE_TAG=branch-main
```
Comment (atarashansky) [inline Dockerfile.cellguide_pipeline:1]: Copied from wmg processing dockerfile with changes for cellguide

File: Makefile [hunk 3]
```diff
@@ -31,6 +31,12 @@ wmg-processing-unittest:
 	DEPLOYMENT_STAGE=test PYTHONWARNINGS=ignore:ResourceWarning coverage run $(COVERAGE_RUN_ARGS) -m pytest \
 	tests/unit/wmg_processing/ --rootdir=. --alluredir=./allure-results --verbose;
 
+.PHONY: cellguide-pipeline-unittest
```
Comment (atarashansky) [inline Makefile:34]: cellguide pipeline unit test command

File: Makefile [hunk 4]
```diff
@@ -109,6 +115,12 @@ local-rebuild-backend: .env.ecr local-ecr-login
 local-rebuild-processing: .env.ecr local-ecr-login
 	docker-compose $(COMPOSE_OPTS) build processing
 
+local-rebuild-wmg-processing: .env.ecr local-ecr-login
```
Comment (atarashansky) [inline Makefile:118]: this didn't exist before, but it should have.

File: docker-compose.yml [hunk 5]
```diff
@@ -251,6 +251,41 @@ services:
       corporanet:
         aliases:
           - processing.corporanet.local
+  cellguide_pipeline:
```
Comment (atarashansky) [inline docker-compose.yml:254]: copied from wmg_processing with changes for cellguide

File: Dockerfile.cellguide_pipeline [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+# For lighter weight Docker images
+ENV PIP_NO_CACHE_DIR=1
+
+WORKDIR /
+
+ADD requirements-cellguide-pipeline.txt requirements-cellguide-pipeline.txt
+RUN pip3 install -r requirements-cellguide-pipeline.txt
+
+ADD backend/__init__.py backend/__init__.py
+ADD backend/wmg/config.py backend/wmg/config.py
```
Comment (joyceyan) [inline Dockerfile.cellguide_pipeline:18]: probably not a big deal, but do we need the wmg files or were they accidentally copied over?

**Review by joyceyan — APPROVED** [review]

File: Dockerfile.cellguide_pipeline (hunk 6 above)
Comment (joyceyan) [reply Dockerfile.cellguide_pipeline:18]: to clarify: i mean this probably isn't a big deal to add the wmg files, but it is maybe a big deal if we forget to add cellguide specific files (that don't exist in this PR, but could exist in future ones)

Comment (atarashansky) [reply Dockerfile.cellguide_pipeline:18]: We need these for the future xD Not in this PR. Specifically, we need to be able to load the snapshot from WMG.

**Pull request merged**